roles/matrix-ldap-registration-proxy/defaults/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+# matrix_ldap_registration_proxy - Want to build a large-scale Matrix server using external registration on LDAP?
+# Project source code URL: https://gitlab.com/activism.international/matrix_ldap_registration_proxy
+
+matrix_ldap_registration_proxy_enabled: true
+
+matrix_ldap_registration_proxy_container_image_self_build_repo: "https://gitlab.com/activism.international/matrix_ldap_registration_proxy.git"
+matrix_ldap_registration_proxy_container_image_self_build_branch: "{{ matrix_ldap_registration_proxy_version }}"
+
+matrix_ldap_registration_proxy_docker_image: "{{ matrix_ldap_registration_proxy_docker_image_name_prefix }}activism.international/matrix_ldap_registration_proxy:{{ matrix_ldap_registration_proxy_version }}"
+matrix_ldap_registration_proxy_docker_image_name_prefix: "localhost/"
+
+matrix_ldap_registration_proxy_version: "296246afc6a9b3105e67fcf6621cf05ebc74b873"
+
+matrix_ldap_registration_proxy_base_path: "{{ matrix_base_data_path }}/matrix_ldap_registration_proxy"
+# We need the docker src directory to be named matrix_ldap_registration_proxy.
+matrix_ldap_registration_proxy_docker_src_files_path: "{{ matrix_ldap_registration_proxy_base_path }}/docker-src/matrix_ldap_registration_proxy"
+matrix_ldap_registration_proxy_config_path: "{{ matrix_ldap_registration_proxy_base_path }}/config"
+
+matrix_ldap_registration_proxy_appservice_public_enabled: false
+
+matrix_ldap_registration_proxy_ldap_uri: ""
+matrix_ldap_registration_proxy_ldap_base_dn: ""
+matrix_ldap_registration_proxy_ldap_user: ""
+matrix_ldap_registration_proxy_ldap_password: ""
+matrix_ldap_registration_proxy_matrix_server_name: "{{ matrix_domain }}"
+matrix_ldap_registration_proxy_matrix_server_url: "https://{{ matrix_server_fqn_matrix }}"
+
+# Controls whether the self-check feature should validate SSL certificates.
+matrix_ldap_registration_proxy_self_check_validate_certificates: true
+
+matrix_ldap_registration_proxy_container_port: 8080
+# Controls whether the matrix_ldap_registration_proxy container exposes its HTTP port (tcp/{{ matrix_ldap_registration_proxy_container_port }} in the container).
+#
+# Takes an "<ip>:<port>" or "<port>" value (e.g. "127.0.0.1:8080"), or empty string to not expose.
+matrix_ldap_registration_proxy_container_http_host_bind_port: ''
+
+matrix_ldap_registration_proxy_registration_addr_with_container: "matrix-ldap_-egistration-proxy:{{ matrix_ldap_registration_proxy_container_port }}"
+matrix_ldap_registration_proxy_registration_addr_sans_container: "127.0.0.1:{{ matrix_ldap_registration_proxy_container_port }}"
```

Comment:
This looks incorrect.

It should probably use the port value from `matrix_ldap_registration_proxy_container_http_host_bind_port` instead of `matrix_ldap_registration_proxy_container_port`.

You can't use `matrix_ldap_registration_proxy_container_http_host_bind_port` directly, because it may contain a port value (e.g. `1234`), but it may also contain an IP address + port value (e.g. `127.0.0.1:1234`). We probably need an additional variable `matrix_ldap_registration_proxy_container_http_host_bind_port_number_raw`. See `matrix_prometheus_node_exporter_container_http_host_bind_port_number_raw` (defined in `roles/matrix-prometheus-node-exporter/vars/main.yml`) for an example: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/5f9f891322effa21d81c468cdcde701b6ba8c72c/roles/matrix-prometheus-node-exporter/vars/main.yml#L1-L5